test_scripts/Polices/External_UCS/028_ATF_P_TC_LPT_Update_PreloadedPT_with_disallowed_by_external_consent_entities_on_struct.lua

```diff
@@ -0,0 +1,146 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: PreloadedPT with "disallowed_by_external_consent_entities_on" struct
+--
+-- Description:
+-- In case:
+-- SDL uploads PreloadedPolicyTable with "disallowed_by_external_consent_entities_on:
+-- [entityType: <Integer>, entityId: <Integer>]" -> of "<functional grouping>" -> from "functional_groupings" section
+-- SDL must:
+-- a. consider this PreloadedPT as valid (with the pre-conditions of all other valid PreloadedPT content)
+-- b. add this "disallowed_by_external_consent_entities_on: [entityType: <Integer>, entityId: <Integer>]"
+-- field of the corresponding "<functional grouping>" in the Policies database.
+--
+-- Preconditions:
+-- 1. Stop SDL (Ignition Off)
```

Comment:
@dboltovskyi, please be sure that in the sdl_preloaded_pt.json for first ignition cycle section disallowed_by_external_consent_entities_off is not observed.